package.json

```diff
@@ -52,13 +52,17 @@
     ]
   },
   "devDependencies": {
-    "@testing-library/react": "^8.0.7",
-    "audit-ci": "^2.3.0",
+    "@testing-library/jest-dom": "^5.1.1",
+    "@testing-library/react": "^8.0.9",
+    "audit-ci": "^2.5.1",
     "husky": "^2.7.0",
-    "jest-dom": "^3.5.0",
     "prettier": "1.18.2",
     "pretty-quick": "^1.11.1",
-    "react-test-renderer": "^16.8.6"
+    "react-scripts": "^3.4.0",
+    "react-test-renderer": "^16.13.0"
+  },
+  "resolutions": {
+    "acorn": "^6.4.1"
   },
```

Comment:
does this just ensure acorn is higher than this version wherever it is used? Could this have unintended consequences if a current package is using a different major version of Acorn?

Reply:
yes it could; this shouldn't be a long term fix. I don'w know if there is a way to explicitly disallow some versions. 

See also this specific note: https://github.com/facebook/create-react-app/issues/8615#issuecomment-597075380

Reply:
I actually just had to deal with this for auth-service, and NPM audit gave me this message

```
# Run  npm update acorn --depth 3  to resolve 1 vulnerability
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ acorn                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ eslint [dev]                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ eslint > espree > acorn                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1488                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

so I ran 

```
npm update acorn --depth 3
```

Which only updated the package-lock.json file. Are you able to do something like that?